### PR TITLE
Add openmmdl

### PR DESCRIPTION
**Pull request by talagayev**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by talagayev** [comment]
@NDoering99 would you want to be also a maintainer of the package?

**Comment by talagayev** [comment]
@hmacdope here is as discussed the conda-forge for my package. Could you check if it makes sense with the version packages and if it is correct.

When I run `pip check` I get this result:

`ipywidgets 7.8.1 requires ipython-genutils, which is not installed.`
`plip 2.3.0 requires openbabel, which is not installed.`
`cuda-python 11.8.2 requires cython, which is not installed.`
`pytest-shutil 1.7.0 requires path-py, which is not installed.`

Is it problematic considering that the package does run even without those packages?

**Comment by NDoering99** [comment]
> @NDoering99 would you want to be also a maintainer of the package?

Yes, I would be happy to act as a package maintainer as well :)

**Comment by talagayev** [comment]
@conda-forge/help-python the package would be ready now to be reviewed.

The Windows and Mac OS Fail due to the packages used, so that it will currently only work for linux based systems.

The pip check leads to errors connected to the version of openbabel, which is not recognized in pip check but is present in conda list. Is there any way to fix this error?

File: recipes/openmmdl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wolberlab/OpenMMDL/archive/refs/tags/{{ version }}.zip
+  sha256: 2504461bed6915d92fa893520a3c9a24c2a1f269112914ffe1d686f5f71a85aa
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/openmmdl/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

**Pull request merged**